### DISCUSSION TITLE
Preview mode issues and questions

### DISCUSSION THREAD

**Discussion by Fierce-Ninja** [zsviczian/obsidian-excalidraw-plugin#198] [Q&A]
Hi there, I have couple of questions/clarifications regarding Preview mode.

1. It would be fantastic if I could have a setting to change default open mode of Excalidraw note to preview mode (just like a regular Obsidian note has one).  Is that possible?
2. When the excalidraw note is first opened, the cursor loses focus.  That means for me to type anything using my keyboard, I first have to use the mouse to activate the note window again.  Is that a bug?  Anything I can do to prevent?

P.S. love your plugin.  Donated couple of times to "buy coffee" link. Thank you!

**Comment by YesterKo** [comment]
Hello!
I think it would be better to mark it as issues, because I don't think there are any solutions to the problems you mentioned unfortunately.

You can make an issue simply by selecting "Create issue from discussion" on the right sidebar, though I'd recommend making seperate issues for the problems.

**Comment by zsviczian** [comment]
The first sounds like #120
I'll prioritize it higher.
